app/controllers/labels_controller.rb

```diff
@@ -0,0 +1,74 @@
+class LabelsController < ApplicationController
+  before_action :set_label, only: [:show, :edit, :update, :destroy]
+
+  # GET /labels
+  # GET /labels.json
+  def index
+    @labels = Label.all
+  end
+
+  # GET /labels/1
+  # GET /labels/1.json
+  def show
+  end
```

Comment:
中身のないメソッドは1行で記載できます。このように書くことで、中身を書き忘れていないことを明示できます。
```suggestion
  def show; end
```

Reply:
全てのコントローラーの中身のないメソッドは1行で記載しました。